### ISSUE TITLE
formatter_utils.dart - toCurrencyString not consistent when deducting/using separators

### ISSUE THREAD

**Issue by igor00krnic** [caseyryan/flutter_multi_formatter#108]
Please consider the code below. 
Int 1234 is formatted like '1.234', when given back to this method the result should remain the same, but the actual result is '1'.

```
toCurrencyString(
  '1.234',
  thousandSeparator: ThousandSeparator.Period,
  mantissaLength: 0,
);
```

I think that I see a couple of problems in this example:
2. When splitting the string to even and fractional part it uses the hardcoded '.' separator. Should it use the separators deduced at the beginning of the mehod (mSeparator, tSeparator)?
3. I think that the fractionalSeparator variable is wrongly deduced. What is the use case of this separator?

**Comment by caseyryan** [comment]
Yeah, that's weird. I'll take a look into it

**Comment by caseyryan** [comment]
Fixed in 2.9.14
